keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -931,13 +1103,14 @@ Pick one more of these and delete the rest.
   - Metric name: apiserver_cache_watcher_buffer_length (histogram, what was the buffer size)
   - Metric name: apiserver_watch_cache_lag (histogram, for how far the cache is behind the expected RV)
   - Metric name: apiserver_terminated_watchers_total (counter, already defined, needs to be updated (by an attribute) so that we count closed watch requests due to an overfull buffer in the new mode)
+  - Metric name: apiserver_watch_list_duration_seconds (histogram, measures latency of watch-list requests)
```

Comment:
1) apiserver_cache_watcher_buffer_length - we don't have such metri, let's remove

2) apiserver_watch_cache_lag   - such metric doesn't exist and is hard to define correcty - let's remove it

Reply:
ok, I thought we would like to list metrics that will be added not the one that already exist.
BTW: why do you think it will be hard to define `apiserver_watch_cache_lag`?

Reply:
> lag vs what

How far a watch request served from the `watchcache` is behind the etcd when it comes to comparing `RVs`.
When I defined that metrics I though it could be useful and easy to compute. I haven't though deeply about it.

Reply:
Difference in RVs isn't particularly useful - if the difference is say 1000 you don't know if there were 1000 objects in the meantime if there weren't anything in the meantime [because only objects of other types were changed].
Also - you would like to ask etcd for current RV, which also isn't particularly great.